deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr.yaml

```diff
@@ -3,10 +3,27 @@ kind: IBMBlockCSI
 metadata:
   name: ibm-block-csi
   namespace: kube-system
+  labels:
+    app.kubernetes.io/name: ibm-block-csi-operator
+    app.kubernetes.io/instance: ibm-block-csi-operator
+    app.kubernetes.io/managed-by: ibm-block-csi-operator
 spec:
   controller:
     repository: ibmcom/ibm-block-csi-driver-controller
     tag: "1.0.0"
   node:
     repository: ibmcom/ibm-block-csi-driver-node
     tag: "1.0.0"
+  sidecars:
```

Comment:
why not
controller:
    csi-controller:
         repo...
         tag...
     sidecar1
     sidecar2
    ....
    csi-node:
        csi-node:
           ...
           ...
        sidecar1(with the full name of cause)


What do you think?  so there is logic between the side cars

Reply:
for some reasons:
1. Flatter and simpler structure
2. For compatibility reasons, we may add new componts except controller and node, add or remove some of the sidecars, even redesign the controller and node components.
3. Less input, like the liveness probe, we don't need to specify it twice.

Reply:
?